docs/functional-tests.md

```diff
@@ -23,12 +24,19 @@
 | Basic VM import with invalid oVirt password should fail import | &check; |
 | Basic VM import with non-existing VM ID should fail import | &check; |
 | Basic VM import with missing specified external resource mapping should fail import | &check; |
+| Basic VM import should fail when source VM does not shut down | &cross; |
 
 ## Basic Net VM
 | Test description | Implemented |
 | :---------------- | :-----------: |
 | Networked VM import should create started VM | &check; |
 
+### Multiple VMs import
+| Test description | Implemented |
+| :---------------- | :-----------: |
+| Two VMs should be imported in parallel to the same namespace and started | &cross; |
+| Two VMs should be imported in sequence to the same namespace and started | &cross; |
```

Comment:
Could you add also a case for importing the same VM twice in sequence ?

Reply:
:+1:

Reply:
Done